frontend/src/metabase/visualizations/click-actions/Mode/Mode.ts

```diff
@@ -32,8 +33,11 @@ export class Mode {
     const mode = this._queryMode;
     const question = this._question;
     const props = { question, settings, clicked, extraData };
+
     const actions = [
-      ...(mode.hasDrills ? queryDrill(question, clicked) : []),
+      ...(mode.hasDrills
+        ? queryDrill(question, clicked, this.isDrillEnabled)
```

Comment:
I'd suggest making names consistent. Here we have `isDrillEnabled` and inside `queryDrill` it's called `availableDrillsFilter`